### PR TITLE
Bugfix: getApp() in sessionAuth()

### PR DESCRIPTION
**Pull request by rabuzarus**
`$a` was not initialized

### CODE REVIEW

File: src/Module/Login.php [hunk 1]
```diff
@@ -1,306 +1,307 @@
...
+		}
+
+		if (isset($_SESSION) && x($_SESSION, 'authenticated')) {
+			if (x($_SESSION, 'visitor_id') && !x($_SESSION, 'uid')) {
+				$r = q("SELECT * FROM `contact` WHERE `id` = %d LIMIT 1",
+					intval($_SESSION['visitor_id'])
+				);
+				if (DBM::is_result($r)) {
+					$a = self::getApp();
+					$a->contact = $r[0];
```
Comment (MrPetovan) [inline src/Module/Login.php]: Please use `self::getApp()->contact` instead.

File: src/Module/Login.php [hunk 2]
```diff
@@ -181,8 +181,7 @@ public static function sessionAuth()
 					intval($_SESSION['visitor_id'])
 				);
 				if (DBM::is_result($r)) {
-					$a = self::getApp();
-					$a->contact = $r[0];
+					self::getApp()->contact;
```
Comment (MrPetovan) [inline src/Module/Login.php]: You left the variable assignment out, I’ll be more careful with my comment next time 😅

Comment (rabuzarus) [reply src/Module/Login.php]: ohhh embarrassing.

File: src/Module/Login.php [hunk 3]
```diff
@@ -181,7 +181,8 @@ public static function sessionAuth()
 					intval($_SESSION['visitor_id'])
 				);
 				if (DBM::is_result($r)) {
-					self::getApp()->contact;
+					$a = self::getApp();
```
Comment (MrPetovan) [inline src/Module/Login.php]: You left the unused $a!

**Comment by annando** [comment]
@rabuzarus It seems as if your editor has changed the line endings to possibly the DOS endings.

**Comment by rabuzarus** [comment]
Since I don't have changed any settings in my editor I don't know how this could happen.

Oh what a crazy day. So many commits for just one line of code. Hopefully this was the last commit.

**Pull request merged**